Content.Server/Sticky/Components/StickyComponent.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Robust.Server.GameObjects;
+
+namespace Content.Server.Sticky.Components;
+using Content.Shared.DrawDepth;
+
+/// <summary>
+///     Items that can be stick to other structures or entities.
+///     For example paper stickers or C4 charges.
+/// </summary>
+[RegisterComponent]
+public sealed class StickyComponent : Component
```

Comment:
im not sure how i feel about it being universally stickable to anything, feels like it should have a whitelist? (or at least an optional whitelist and we can tune it later if it becomes an issue)